README.md

```diff
@@ -64,10 +64,10 @@
 ### Setting Up Locally
 
 - Install [Go](https://go.dev/).
+- Install [Docker](https://docs.docker.com/engine/install/).
```

Comment:
Can you also add the steps required to set up the db locally?

Reply:
Also replace SQLiteStudio with a postgres equivalent.

Reply:
> Also replace SQLiteStudio with a postgres equivalent.
Not really sure what would be a proper equivalent. Can find several options on google.

Reply:
You can list multiple options. They have to be free and open source and free of cost, though. Being easy to install and use would be a nice to have.

Reply:
A desktop app would be best since it will have minimal setup.